tephra-hbase-compat-0.98/src/test/java/co/cask/tephra/hbase98/TransactionAwareHTableTest.java

```diff
@@ -363,6 +363,34 @@ public void testAbortedTransactionalDelete() throws Exception {
   }
 
   @Test
+  public void testColumnFamilyDelete() throws Exception {
```

Comment:
It would be a good idea to call this `testDeleteRollback()`, since it tests both row and family delete, right?